### DISCUSSION TITLE
Can the question mark (?) be placed on the same line as the question.

### DISCUSSION THREAD

**Discussion by buzhiming999** [st3v3nmw/obsidian-spaced-repetition#831] [Q&A]
?    standalone line doesn't look very nice.

Question A
?
Answerline1
Answerline2
Answerline3

Question A  ?
Answerline1
Answerline2
Answerline3

**Comment by ronzulu** [comment]
Hi @buzhiming999 sorry but the code doesn't support your preferred format.

However you can change the "?" to something else that might find nicer.

![image](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/75528127/6b69db1b-fabd-4078-a805-839d4f0fe438)

Cheers
Ronny

**Comment by ronzulu** [comment]
FYI there is another suggestion regarding question/answer formats:
[FEAT] Support Q and A style https://github.com/st3v3nmw/obsidian-spaced-repetition/issues/714

**Reply by buzhiming999** [reply]
Thank you for your enthusiastic response. Although a solution is not currently available, it may be implemented in future updates.

**Comment by mccann** [comment]
I suspect you wanted your Obsidian notes to look like reasonable documents, as well as be used as flash cards, and this is why you found the ``?`` on a single line ugly. 

As a workaround, I am taking advantage of markdown comment syntax for my multiline card separator, after configuring spaced-repetition to use ``%%flashcard%%``, I can write the following:

```
Multiline Question
%%flashcard%%
Multiline Answer
```

This is not ideal, but it does mean that the rendered note doesn't have an errant ``?`` floating.  
Unfortunately it introduced a blank line in some cases. If the answer is a list or quote, the blank line is collapsed.
